roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +36,33 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
-- name: start netmaster
-  shell: systemctl daemon-reload && systemctl start netmaster
+- name: restart netmaster
+  shell: systemctl daemon-reload && systemctl stop netmaster && systemctl start netmaster
   when: run_as == "master"
 
 - name: copy environment file for netplugin
-  copy: src=netplugin dest=/etc/default/netplugin
+  template: src=netplugin.j2 dest=/etc/default/netplugin mode=0644
 
 - name: copy systemd units for netplugin
   copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
 
-- name: start netplugin
-  shell: systemctl daemon-reload && systemctl start netplugin
+- name: restart netplugin
+  shell: systemctl daemon-reload && systemctl stop netplugin && systemctl start netplugin
 
 - name: setup netmaster host alias
   shell: echo "{{ service_vip }} netmaster" >> /etc/hosts
 
 # XXX: need to move the following to correct roles
-- name: install contivctl
-  shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
-
-- name: install docker-compose
-  shell: pip install docker-compose
-
-- name: install contiv-compose
-  shell: >
-     source /etc/profile.d/00golang.sh && \
-     go get github.com/contiv/libcompose && \
-     cd $GOPATH/src/github.com/contiv/libcompose && \
-     make binary && \
-     ln -s `pwd`/bundles/libcompose-cli_linux-amd64 $GOPATH/bin/contiv-compose
+#- name: install contivctl
```

Comment:
if we're going to comment this code out, I would prefer we remove it. We can get it from source control later.

fwiw @mapuri, @jainvipin building and installing the contiv-compose tool takes enough time for ansible to timeout.

We could use the ansible async stuff but this is the best answer I could find regarding a general timeout: http://grokbase.com/t/gg/ansible-project/149fts2xd2/how-to-set-timeout-for-shell-command which is not very promising. 

What I think we need to do is improve build performance before we can install this. I think we can come up with some good ways to do this such as building images with godeps preinstalled, or inventing release tooling through travis or whatnot for contiv-compose.

So my suggestion here is that we just remove this code for now and restore it later.